StereoKitC/stereokit.h

```diff
@@ -1535,6 +1535,8 @@ SK_API void                  render_add_mesh       (mesh_t mesh, material_t mate
 SK_API void                  render_add_model      (model_t model, const sk_ref(matrix) transform, color128 color_linear sk_default({1,1,1,1}), render_layer_ layer sk_default(render_layer_0));
 SK_API void                  render_blit           (tex_t to_rendertarget, material_t material);
 SK_API void                  render_screenshot     (const char *file, vec3 from_viewpt, vec3 at, int32_t width, int32_t height, float field_of_view_degrees);
+SK_API void                  render_screenshot_capture(void (*render_on_screenshot_callback)(uint8_t* out_arr_data, int32_t data_length), vec3 from_viewpt, vec3 at, int32_t width, int32_t height, float field_of_view_degrees);
```

Comment:
What is the context used for?

Reply:
Since C style callbacks don't "capture" the variables around them like C# callbacks do, so this is a way of passing in the "context" around that callback. For example, with a write-to-file callback, the context may include the string for the file path!

Reply:
Ahh, so it offers that flexibility even if it's unused. Neat!